datasource/opennms-direct/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -1,12 +1,33 @@
-<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0" >
+
+    <cm:property-placeholder id="datasourceProperties" persistent-id="org.opennms.oce.datasource.opennms.direct" update-strategy="reload">
+        <cm:default-properties>
+            <cm:property name="scriptFile" value="" />
+            <cm:property name="scriptCacheMillis" value="30000"/>  <!-- 30 seconds -->
+        </cm:default-properties>
+    </cm:property-placeholder>
+
     <reference id="alarmDao" interface="org.opennms.integration.api.v1.dao.AlarmDao" />
     <reference id="alarmFeedbackDao" interface="org.opennms.integration.api.v1.dao.AlarmFeedbackDao" />
     <reference id="nodeDao" interface="org.opennms.integration.api.v1.dao.NodeDao" />
     <reference id="eventForwarder" interface="org.opennms.integration.api.v1.events.EventForwarder" />
 
+    <bean id="scriptFactory" class="org.opennms.oce.datasource.opennms.jvm.ScriptedInventoryImpl" init-method="init" >
```

Comment:
Should this be "scriptService" instead? Same for below. Doesn't look like this is a factory.